custom_components/tuya_local/devices/smartmi_e1_purifier.yaml

```diff
@@ -0,0 +1,61 @@
+name: Smartmi Air Purifier E1
```

Comment:
Just make this "Air purifier", in case other brands match the same config.

Reply:
I looked in other configuration files and the brand is indicated everywhere in them. For example, "Breville Smart Air Connect Purifier" or "Himox H06 air purifier". Is it really worth removing the brand name here?

Reply:
Yes, there are a lot of old configs that have not been updated, but that does not mean new ones should not follow new naming rules that have evolved through experience.